These patches allow the man pages to be built using older version
of Sphinx. These patches may be dropped for more recent versions.
Currently, our man pages are built on submit-3.batlab.org and
python-sphinx10 is installed from EPEL 6. (version 1.0.8)
These components are not used for the man pages.

* sphnix.ext.napoleon appears in version 1.3
* sphinx.ext.autosectionlabel appears in version 1.4

diff --git a/docs/conf.py b/docs/conf.py
index eb88a06..2da06e6 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -32,10 +32,8 @@ sys.path.append(os.path.abspath('extensions'))
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
-    'sphinx.ext.autosectionlabel',
     'sphinx.ext.intersphinx',
     'sphinx.ext.autodoc',
-    'sphinx.ext.napoleon',
     'ticket',
     'macro',
     'index',
